pages/api/benefits.js

```diff
@@ -62,13 +62,22 @@ const isOver60 = (situation) => {
   return eligiblityStatus.NOT_ELIGIBLE;
 };
 
+const isOver65 = (situation) => {
+  if (!situation["age"]) return eligiblityStatus.POTENTIALLY_ELIGIBLE;
+  else if (situation["age"] >= 65) return eligiblityStatus.ELIGIBLE;
+  return eligiblityStatus.NOT_ELIGIBLE;
+};
+
 const isUnder65 = (situation) => {
   if (!situation["age"]) return eligiblityStatus.POTENTIALLY_ELIGIBLE;
   else if (situation["age"] < 65) return eligiblityStatus.ELIGIBLE;
   return eligiblityStatus.NOT_ELIGIBLE;
 };
 
 const eligibilityCriteria = {
+  oas: [isOver65],
   alw: [isOver60, isUnder65],
+  alws: [isOver60, isUnder65],
   cpp: [isOver60],
+  "cpp-gis": [isOver65],
 };
```

Comment:
I actually like how this is set up, it's easy to keep track of eligibility like this!